app/models/deploy_group.rb

```diff
@@ -28,6 +28,10 @@ def long_name
     "#{name} (#{environment.name})"
   end
 
+  def natural_order
+    name.split(/(\d+)/).each_with_index.map { |x, i| i.odd? ? x.to_i : x }
```

Comment:
would only work if all had a number in their name
something like master1 / pod1 should still sort alphabetically

On Mon, Apr 11, 2016 at 9:19 PM, Roman Sandler <notifications@github.com>
wrote:

> In app/models/deploy_group.rb
> <https://github.com/zendesk/samson/pull/811#discussion_r59318814>:
>
> > @@ -28,6 +28,10 @@ def long_name
> >      "#{name} (#{environment.name})"
> >    end
> >
> > +  def natural_order
> > +    name.split(/(\d+)/).each_with_index.map { |x, i| i.odd? ? x.to_i : x }
>
> name[/\d+/].to_i ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/811/files/aa3223a4f9c3f364dc8e2383c2168d3a0cdf7679#r59318814>
>